corehq/apps/hqmedia/models.py

```diff
@@ -672,6 +672,11 @@ def rename_media(self, old_path, new_path):
         if hasattr(self, 'case_list') and self.case_list.show:
             count += self.rename_menu_media(self.case_list, old_path, new_path)
 
+        # Case search and claim menu items
+        if hasattr(self, 'search_config'):
+            count += self.rename_menu_media(self.search_config.search_label, old_path, new_path)
+            count += self.rename_menu_media(self.search_config.search_again_label, old_path, new_path)
```

Comment:
accidentally stumbled on this method while searching for `hasattr(self, 'case_list')` :)

Reply:
Seems like it's part of ICDS's multimedia paths upload/download. I'm tempted to say we should remove that whole feature, but it looks like USH is using it.